core/src/main/scala/kafka/server/ClientQuotaManager.scala

```diff
@@ -220,6 +216,14 @@ class ClientQuotaManager(private val config: ClientQuotaManagerConfig,
     ClientSensors(quotaSensor, throttleTimeSensor)
   }
 
+  private def getThrottleTimeSensorName(clientId: String): String = {
+    apiKey + "ThrottleTime-" + clientId
+  }
+
+  private def getQuotaSensorName(clientId: String): String = {
+    apiKey + "-" + clientId
+  }
+
```

Comment:
A very minor comment here about style -- when we write one line function in scala in kafka, do we prefer to avoid "{}" and implement function in one line?